src/main.js

```diff
@@ -261,6 +266,24 @@ function updateNotifications(app) {
 
         processNotificationCount(app, sum);
     });
+
+    // The following is a workaround for the Electron bug where after an indeterminate
+    // period of inactivity, the main application window turns into a blank white screen.
+    // When this happens, inspection shows that the loaded page consists of the following
+    // empty HTML markup:
+    //
+    //     <html><head></head><body></body></html>
+    //
+    // As such, we perform a simple check as to whether the <body/> of our loaded page
+    // has become empty.  If it has, then we automatically reload Google Voice for the
+    // user.  This seems to eliminate the problem entirely, without any adverse effects,
+    // as once we detect an empty body, the application is already in a non-working state.
+    win.webContents.executeJavaScript("document.getElementsByTagName('body')[0].innerText.trim()").then(
```

Comment:
Just had one more thought. Instead of stringifying the contents of body every three seconds can we just check the body's node count for 0?

Reply:
Slick!  Let me run with those changes in place today and make sure it still works.  Seems like it should.